curvefs/src/metaserver/inode_manager.cpp

```diff
@@ -316,6 +316,24 @@ MetaStatusCode InodeManager::UpdateInode(const UpdateInodeRequest& request,
         ++(*deletedNum);
     }
 
+    const S3ChunkInfoMap &map2add = request.s3chunkinfoadd();
```

Comment:
why add this map, and it not exist in before?

Reply:
You can look at the GetOrModifyS3ChunkInfoRequest interface to request the map that has been used all the time, here it is just merged into UpdateInodeRequest